dataeng/resources/warehouse-transforms-master-ci.sh

```diff
@@ -50,7 +50,7 @@ then
     # Schema is dynamically created against each run.
     # profiles.yml contains the name of Schema which is used to create output models when dbt runs. 
     python create_ci_schema.py --key_path $KEY_PATH --passphrase_path $PASSPHRASE_PATH --automation_user $USER --account $ACCOUNT --db_name $DB_NAME --schema_name $CI_SCHEMA_NAME 
-
+    sleep 10s
```

Comment:
Can you also add a comment in the shell script, noting why you are doing it?

Reply:
yes sure, added